src/scripts/underground/Underground.ts

```diff
@@ -88,6 +112,22 @@ class Underground implements Feature {
         return Underground.BASE_BOMB_EFFICIENCY + this.getUpgrade(UndergroundUpgrade.Upgrades.Bomb_Efficiency).calculateBonus();
     }
 
+    getSurveyCharge_Cost() {
+        return Underground.PROSPECT_ENERGY - this.getUpgrade(UndergroundUpgrade.Upgrades.SurveyCharge_Cost).calculateBonus();
+    }
+
+    getSurveyCharge_Efficiency() {
```

Comment:
Can we stick to camelCase here, and also `Survey Charge` → `Survey` or `Prospect`
```suggestion
    getSurveyCost() {
        return Underground.PROSPECT_ENERGY - this.getUpgrade(UndergroundUpgrade.Upgrades.SurveyCharge_Cost).calculateBonus();
    }

    getSurveyEfficiency() {
```

Reply:
Done. Changed to just Survey.